File: checkoutsys-api/checkoutsys.api/checkoutsys.api/Models/Order.cs
using System;
using System.Collections.Generic;

namespace checkoutsys.api.Models;

public partial class Order
{
    public long Id { get; set; }

    public long CustomerId { get; set; }

    public DateTime Date { get; set; }

    public string IsCompleted { get; set; } = null!;

    public virtual User Customer { get; set; } = null!;

    public virtual ICollection<OrdersProduct> OrdersProducts { get; } = new List<OrdersProduct>();

    public virtual Transaction? Transaction { get; set; }
}

File: checkoutsys-ui/checkoutsys.ui/src/app/models/order.ts
import { OrderProducts } from "./order-products";
import { Transaction } from "./transaction";
import { User } from "./user";

export class Order {
    id: number = 0;
    customerId: number = 0;
    date: Date = new Date();
    isCompleted = "";
    user?: User;
    orderProducts: OrderProducts[] = [];
    transaction?: Transaction;
}